tests/test_middleware.py

```diff
@@ -220,3 +220,30 @@ def is_revoked(request, payload):
     })
     assert response.status == 403
     assert 'Token is revoked' in response.reason
+
+
+@pytest.mark.parametrize(
+    "configured_header_prefix, provided_header_prefix, resp_status", [
+        ("Bearer", "Bearer", 200),
```

Comment:
This two could be dropped as well. This behaviour already covered, just test custom value provided.